source/includes/_relationships.md

```diff
@@ -0,0 +1,69 @@
+# Relationships
+```shell
+# EXAMPLE
+GET https://api.prospect.io/public/v1/prospects/?include=organization
+```
+
+> Get prospects with their associated organizations
+
+Resources returned by the API are returned in the JSON API format, which means that related entities are 
+linked using a "relationship" section in the JSON payloads. 
+This section only contains the links between the unique identifier of related records (in the example, Prospect#29964 and Organization#1). 
+
+In order to get access to associated information on an entity, you can use the parameter `?include=`, which will create 
```

Comment:
What if want many relationships to be included, i.e. prospect's creator and organization. Can I do `?include=creator,organization`? This needs to be precised.